### DISCUSSION TITLE
Close browser tab on click or cause an error "Scripts may close only the windows that were opened by them. [Alpine] ! $refs.panel.contains($event.target) && close() @ VM29:3""

### DISCUSSION THREAD

**Discussion by MayaSol** [alpinejs/alpine#4274] [7. Bugs (archived)]
I have a page with alpine.js.
In some browsers tab closes when clicking on any button on page. (Yandex Browser)
In others browsers an error appears (Chrome, Firefox):
"Scripts may close only the windows that were opened by them.
 [Alpine] ! $refs.panel.contains($event.target) && close() @ VM29:3""
 
 An error reproduces on http://atlonfm.com
 
![Selection_167](https://github.com/alpinejs/alpine/assets/6613669/b449d54b-47f2-46cd-b501-a7ee1049b72d)

**Comment by MayaSol** [comment]
It was error in my code 
x-on:focusin.window="! $refs.panel.contains($event.target) && close()" assumes close() as window.close()

**Comment by SimoTod** [comment]
You can't do it because of security restrictions (https://stackoverflow.com/questions/19761241/window-close-and-self-close-do-not-close-the-window-in-chrome#19768082). It's your browser stopping you, there's nothing Alpine (or any other js framework) can do for it.
